azure/azure-release-pipeline.yml

```diff
@@ -114,6 +114,22 @@ extends:
           smoke_tests: true
       depends_on:
       - manual_approval_int
+    - environment: manual-approval
+      stage_name: manual_approval_dep
+    - environment: dev
+      stage_name: ers_dep_release
+      service_name: "${{ variables.service_name }}-dep"
+      short_service_name: "${{ variables.short_service_name }}-dep"
+      service_base_path: "${{ variables.service_base_path }}-dep"
+      enable_status_monitoring: false
```

Comment:
I think we turned status monitoring off for FIX as this environment is only turned on when needed. This resulted in lots of status errors.

I suspect DEP env is available all the time so it is safe to leave status monitoring on (the default). Maybe check with @stuandgray around env availability.

Reply:
"The Spine CIS **Deployment** environment has been restored to full 24 hour, 7 days a week availability and will remain fully available until further notice". - [https://digital.nhs.uk/services/path-to-live-environments/path-to-live-environment-news/news---week-commencing-23-january-2023 ](url)